### PR TITLE
platform specific default paths

### PR DESCRIPTION
**Pull request by vbatts**
Isolate platform specific paths into variables

### CODE REVIEW

**Comment by rhatdan** [comment]
LGTM

File: lib/config.go [hunk 1]
```diff
@@ -11,19 +11,13 @@ import (
...
 	pauseImage          = "k8s.gcr.io/pause:3.1"
 	pauseCommand        = "/pause"
 	defaultTransport    = "docker://"
-	seccompProfilePath  = "/etc/crio/seccomp.json"
 	apparmorProfileName = "crio-default"
-	cniConfigDir        = "/etc/cni/net.d/"
-	cniBinDir           = "/opt/cni/bin/"
 	cgroupManager       = oci.CgroupfsCgroupsManager
-	lockPath            = "/run/crio.lock"
-	containerExitsDir   = oci.ContainerExitsDir
```
Comment (wking) [inline lib/config.go]: This can stay right?  You're also adjusting the oci package in this commit to get an OS-appropriate `ContainerExitsDir` value.

**Comment by wking** [comment]
Checking the Travis errors, it looks like [your missing a Signed-off-by][1] and that [there are a bunch of debug and warn calls with inappropriate formatting][2].  The latter is probably unrelated to this commit and just the existing CRI-O code-base vs. improved linting.

[1]: https://travis-ci.org/kubernetes-incubator/cri-o/jobs/394683423#L1089
[2]: https://travis-ci.org/kubernetes-incubator/cri-o/jobs/394683426#L1108

**Comment by TomSweeneyRedHat** [comment]
LGTM

**Comment by wking** [comment]
All green, although I'd still see [no need to move `containerExitsDir` into per-OS files][1].

[1]: https://github.com/kubernetes-incubator/cri-o/pull/1634#discussion_r196918411

**Pull request merged**